pkg/spanstat/spanstat_test.go

```diff
@@ -182,10 +182,11 @@ func TestSpanStatRaceCondition(t *testing.T) {
 			for i := 0; i < 5; i++ {
 				wg.Add(1)
 				go func(span *SpanStat) {
+					defer wg.Done()
 					assert.NotEqual(t, tt.fields.runFunc(span), float64(0))
 				}(span)
 			}
-			wg.Done()
+			wg.Wait()
```

Comment:
opps, how come I missed this.

Thanks :tada:

Reply:
oh right, I rememberred now. Actually, I was supposed to remove this test, as it didn't fit well with entire test framework in code base (e.g. with TestSuite.C).

Reply:
>  it didn't fit well with entire test framework in code base (e.g. with TestSuite.C).

I think this is fine. These tests are standard table driven tests which we have a ton in `pkg/hubble` for instance.